core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala

```diff
@@ -84,7 +95,14 @@ class BillingProjectOrchestrator(ctx: RawlsRequestContext,
         creationStatus,
         None
       )
-    } yield {}
+    } yield {}).recover {
+      case e: RawlsExceptionWithErrorReport =>
+        e.errorReport.statusCode.collect { case _: ServerError =>
+          tagAndCaptureSentryEvent(e)
+        }
+      case wsmException: WsmApiException if wsmException.getCode >= 500 => tagAndCaptureSentryEvent(wsmException)
+      case bpmException: BpmApiException if bpmException.getCode >= 500 => tagAndCaptureSentryEvent(bpmException)
```

Comment:
There might be a less repetitive way to do this, but it wasn't coming to mind and at least this is pretty clear about which exceptions we want to alert on.

Reply:
There are some ways, but since it's just a simple function call, I think it's good as is